templates/console/source/lambda/config-version/cf_config_version_manager_graphql/cf_config_version_manager_graphql.py

```diff
@@ -694,23 +726,56 @@
             'dateTime': current_time,
             'note': snapShotNote,
         })
-
     return {
         'statusCode': 200,
         'body': 'succeed create new snapshot'
     }
 
 
+def get_latest_version(ddb_client, distributionId):
+    # first get the latest versionId of updated distribution
+    latest_table = ddb_client.Table(DDB_LATESTVERSION_TABLE_NAME)
+    try:
+        resp = latest_table.query(
+            KeyConditionExpression=Key('distributionId').eq(distributionId)
+        )
+        log.info(resp)
+    except Exception as e:
+        logging.error(e)
+    record_list = resp['Items']
+    if len(record_list) == 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0` or `if len(val) &gt; 0`

 [Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)